### PR TITLE
#235 remove wording request

### PR DESCRIPTION
**Pull request by kaydentsoftwire**
## What's changed
'Request denied' changed to 'Parcel Denied' on the Status drop down menu
'Delete Parcel Request' changed to 'Delete Parcel' on the Action drop down menu
'Request denied' and request deleted' changed to 'parcel denied' and 'parcel deleted' in the database via migration file and changes to the seed
Same thing changed when you click on a parcel and see its event history

## Screenshots / Videos
| Before     | After      |
|------------|------------|
| imageimageimage| imageimageimage|

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA
- [x] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x] Make sure you've tested via `npm run test:e2e`

If you have made any changes to the database...
  - [x] The migration files are up-to-date with my final set up (`npx supabase db diff -f <name_of_migration>` should create nothing at this point)
  - [x] I have updated the typescript definitions for the database with `db:local:generate_types`
  - [x] I have modified the seed in `supabase/seed/seed.mts` if appropriate
  - [x] If I have modified the seed, I have also generated the seed with `npm run db:generate_seed` 
  - [x] With my final set up, I can run `npm run dev:reset_supabase` without any errors.
  - Does this require resetting the deployed database? - YES / NO (remove as appropriate)
  - [x] I have checked that migration can happen successfully without resetting the database, doing the following.
    - Make sure you have rebased your branch onto dev or merged dev into your branch.
    - Checkout the dev branch
    - Run `npm run post_checkout` to reset the database, including the seed data.
    - Checkout your branch
    - Run `npx supabase migrations up --include-all --local` to run all outstanding migration files
    - Check that the resulting database is what you expect, bar any seed data changes.

### CODE REVIEW

File: supabase/migrations/20240503131452_rename_request_denied_to_parcel_denied.sql
Comment (kaydentsoftwire) [inline supabase/migrations/20240503131452_rename_request_denied_to_parcel_denied.sql:1]: A duplicate caused because: I wrote it incorrectly the first time and it didn't do anything (I got 'parcel denied' and 'request denied' the wrong way round), then I edited this file but it didn't seem to update - I guess because it wasn't a new migration? - so I added another one. 
Be grateful to hear if there is a better way of dealing with this :)

Comment (minalui) [reply supabase/migrations/20240503131452_rename_request_denied_to_parcel_denied.sql]: @kaydentsoftwire you can just delete the migration file and recreate a new one

Comment (kaydentsoftwire) [reply supabase/migrations/20240503131452_rename_request_denied_to_parcel_denied.sql]: Thank you :)

**Comment by AmeliaIsCoding** [comment]
I think Ricky should review this before we merge because DB changes

**Comment by AmeliaIsCoding** [comment]
Can you fill out the "what's changed" and the screenshots sections of the PR description pls?

File: supabase/migrations/20240503124339_rename_request_deleted_and_request_denied.sql
Comment (AmeliaIsCoding) [inline supabase/migrations/20240503124339_rename_request_deleted_and_request_denied.sql:3]: fyi for Ricky - I told Kay to use on update cascade - i thought this would be good, but aware you might not think this is best

**Review by AmeliaIsCoding — COMMENTED** [review]
Looks good to me, well done. Just think we should wait for Ricky to review this one, bc of DB changes

File: supabase/migrations/20240503124339_rename_request_deleted_and_request_denied.sql
Comment (rickaw) [reply supabase/migrations/20240503124339_rename_request_deleted_and_request_denied.sql:3]: Agree we should `on update cascade`!

**Review by rickaw — APPROVED** [review]

**Pull request merged**
